centreon/www/api/class/centreon_ceip.class.php

```diff
@@ -228,7 +274,7 @@ private function getLicenseInformation(): array
             }
         } catch (\Pimple\Exception\UnknownIdentifierException) {
             // The licence does not exist, 99.99% chance we are on Open source. No need to log.
-        } catch (\Exception $exception) {
+        } catch (\Exception | \Throwable $exception) {
```

Comment:
Throwable is an interface, while Exception is a class that implements Throwable.
Throwable is therefore sufficient.

```suggestion
        } catch (\Throwable $exception) {
```